api-module-library/hubspot/manager.js

```diff
@@ -31,8 +31,9 @@ class Manager extends ModuleManager {
             client_id: process.env.HUBSPOT_CLIENT_ID,
             client_secret: process.env.HUBSPOT_CLIENT_SECRET,
             scope: process.env.HUBSPOT_SCOPE,
-            redirect_uri: `${process.env.REDIRECT_URI}/hubspot`,
+            redirect_uri: process.env.REDIRECT_URI,
```

Comment:
This one is tricky... @MichaelRyanWebber do you have an opinion on how we should navigate this? Each Frigg adopter may have a slightly different way they want to handle OAuth redirects, so if we're going to default to our "base url + api name" convention, we should have an easy override.

Overriding seems like a not so simple thing to do in Frigg v1 unless we have it at the module options level the integration definition?

In the current FreshBooks implementation, @Jo-lefthook, we should probably override the getInstance method in the local HubSpotEntityManager and change the redirect argument there instead of here in the published API Module

Reply:
@seanspeaks Yea, may we can start introducing an env variable for each integration library. for example, redirect_uri here would be something like `process.env.FRIGG_HUBSPOT_REDIRECT_URI || ${process.env.REDIRECT_URI}/hubspot`

Reply:
@Jo-lefthook @seanspeaks  I think we should follow the same pattern as Microsoft Teams, check it out here: 
https://github.com/friggframework/frigg/blob/main/api-module-library/microsoft-teams/manager.js

It allows the user to override the redirect_uri, and if not it gets the default one `${process.env.REDIRECT_URI}/hubspot` already using Frigg's helpers

Reply:
Cool, so we agree on the Microsoft Teams approach. I've pushed a commit implementing that. For the env variable, as @MichaelRyanWebber said, I think this should be added to the pre-release checklist to be implemented in all API modules